src/test/codewhisperer/util/crossFileContextUtil.test.ts

```diff
@@ -49,6 +57,71 @@ describe('crossFileContextUtil', function () {
         })
     })
 
+    describe('getCrossFileCandidate', function () {
+        before(async function () {
+            this.timeout(60000)
```

Comment:
Why does this test need so much time? We should avoid assigning giant timeouts for tests unless they really are doing a lot of work.

Reply:
gotcha, let me remove it, i referred to `visualizeStateMachine.test.ts` and thought this timeout is needed for tests manipulating editos or something lol